app/src/main/java/hm/orz/chaos114/android/tumekyouen/util/EncryptionUtil.java

```diff
@@ -8,108 +11,102 @@
 import javax.crypto.KeyGenerator;
 import javax.crypto.spec.SecretKeySpec;
 
-import android.content.Context;
-import android.util.Base64;
-
 public class EncryptionUtil {
-	/** 鍵のbit数 */
-	private static final int ENCRYPT_KEY_LENGTH = 128;
+    /** 鍵のbit数 */
+    private static final int ENCRYPT_KEY_LENGTH = 128;
+
+    /** 暗号化キー */
+    private final Key key;
 
-	/** 暗号化キー */
-	private final Key key;
+    /**
+     * コンストラクタ。
+     * 暗号化キーを生成、もしくは復元します。
+     *
+     * @param context コンテキスト
+     */
+    public EncryptionUtil(Context context) {
+        context = context.getApplicationContext();
 
-	/** コンテキスト */
-	private final Context context;
+        // Preferenceから暗号化キーを取得
+        PreferenceUtil preferenceUtil = new PreferenceUtil(context);
+        String keyStr = preferenceUtil.getString(PreferenceUtil.KEY_SECRET_KEY);
 
-	/**
-	 * コンストラクタ。
-	 * 暗号化キーを生成、もしくは復元します。
-	 * 
-	 * @param context コンテキスト
-	 */
-	public EncryptionUtil(Context context) {
-		this.context = context.getApplicationContext();
-		
-		// Preferenceから暗号化キーを取得
-		PreferenceUtil preferenceUtil = new PreferenceUtil(this.context);
-		String keyStr = preferenceUtil.getString(PreferenceUtil.KEY_SECRET_KEY);
+        if (keyStr == null) {
+            // Preferenceから取得できなかった場合
 
-		if (keyStr == null) {
-			// Preferenceから取得できなかった場合
+            // 暗号化キーを生成
+            key = generateKey();
+            // 生成したキーを保存
+            String base64Key = Base64.encodeToString(key.getEncoded(),
+                    Base64.URL_SAFE | Base64.NO_WRAP);
+            preferenceUtil.putString(PreferenceUtil.KEY_SECRET_KEY, base64Key);
+        } else {
+            // Preferenceから取得できた場合
 
-			// 暗号化キーを生成
-			key = generateKey();
-			// 生成したキーを保存
-			String base64Key = Base64.encodeToString(key.getEncoded(),
-					Base64.URL_SAFE | Base64.NO_WRAP);
-			preferenceUtil.putString(PreferenceUtil.KEY_SECRET_KEY, base64Key);
-		} else {
-			// Preferenceから取得できた場合
-			
-			// キーを復元
-			byte[] keyBytes = Base64.decode(keyStr, Base64.URL_SAFE
-					| Base64.NO_WRAP);
-			key = new SecretKeySpec(keyBytes, "AES");
-		}
-	}
+            // キーを復元
+            byte[] keyBytes = Base64.decode(keyStr, Base64.URL_SAFE
+                    | Base64.NO_WRAP);
+            key = new SecretKeySpec(keyBytes, "AES");
+        }
+    }
 
-	/**
-	 * 暗号化した文字列を返却する。
-	 * 
-	 * @param input 入力文字列
-	 * @return 暗号化した文字列
-	 */
-	public String encrypt(String input) {
-		if (input == null) {
-			return null;
-		}
+    /**
+     * 暗号化キーを生成する。
+     *
+     * @return 暗号化キー
+     */
+    private static Key generateKey() {
+        try {
+            KeyGenerator generator = KeyGenerator.getInstance("AES");
```

Comment:
WARNING: [TrulyRandom] Potentially insecure random numbers on Android 4.3 and older. Read https://android-developers.blogspot.com/2013/08/some-securerandom-thoughts.html for more info.
 Key generation, signing, encryption, and random number generation may not receive cryptographically strong values due to improper initialization of the underlying PRNG on Android 4.3 and below.

If your application relies on cryptographically secure random number generation you should apply the workaround described in https://android-developers.blogspot.com/2013/08/some-securerandom-thoughts.html .

This lint rule is mostly informational; it does not accurately detect whether cryptographically secure RNG is required, or whether the workaround has already been applied. After reading the blog entry and updating your code if necessary, you can disable this lint issue.